.eslintrc

```diff
@@ -0,0 +1,113 @@
+
+    // Discuss?
+    "new-cap": 1,
+    "no-debugger": 1,
+    "no-fallthrough": 1,
+    "no-inner-declarations": [1, "functions"],
+    "no-loop-func": 1,
+    "no-multi-spaces": 1,
+    "no-native-reassign": 1,
+    "no-negated-in-lhs": 1,
+    "no-nested-ternary": 1,
+    "no-new-func": 1,
+    "no-proto": 1,
+    "object-curly-spacing": [1, "always"],
+    "quote-props": [1, "as-needed"],
+    "radix": 1,
+
+    // Done, but up for discussion
+    "space-after-keywords": [1, "always"],
+    "space-before-blocks": [1, "always"],
+    "space-before-function-paren": [1, "never"],
+
+    // Pretty standard
+    "brace-style": [2, "1tbs"],
+    "camelcase": 2,
+    "comma-dangle": [2, "never"],
+    "comma-spacing": [2, {"before": false, "after": true}],
+    "comma-style": [2, "last"],
+    "computed-property-spacing": [2, "never"],
+    "consistent-this": [1, "self"],
+    "curly": [2, "all"],
+    "default-case": 2,
+    "dot-location": [2, "property"],
+    "dot-notation": [2, { "allowKeywords": true }],
+    "eol-last": 2,
+    // Need to confirm following the proto is never desired
+    "guard-for-in": 1,
+    "handle-callback-err": [2, "^(err|error)$"],
+    "key-spacing": [2, { "beforeColon": false, "afterColon": true }],
+    "new-parens": 2,
+    "no-caller": 2,
+    "no-catch-shadow": 2,
+    "no-control-regex": 2,
+    "no-delete-var": 2,
+    "no-dupe-args": 2,
+    "no-dupe-keys": 2,
+    "no-duplicate-case": 2,
+    "no-else-return": 2,
+    "no-empty": 2,
+    "no-empty-character-class": 2,
+    "no-ex-assign": 2,
+    "no-extend-native": 2,
+    "no-extra-bind": 2,
+    "no-extra-boolean-cast": 2,
+    "no-extra-semi": 2,
+    "no-floating-decimal": 0,
+    "no-func-assign": 2,
+    "no-implied-eval": 2,
+    "no-invalid-regexp": 2,
+    "no-irregular-whitespace": 2,
+    "no-iterator": 2,
+    "no-labels": 2,
+    "no-lone-blocks": 2,
+    "no-lonely-if": 2,
+    "no-mixed-spaces-and-tabs": [2, false],
+    "no-multi-str": 2,
+    "no-multiple-empty-lines": [2, { "max": 1 }],
+    "no-new": 2,
+    "no-new-require": 2,
+    "no-obj-calls": 2,
+    "no-octal": 2,
+    "no-octal-escape": 2,
+    "no-path-concat": 2,
+    "no-redeclare": 2,
+    "no-regex-spaces": 2,
+    "no-reserved-keys": 2,
+    "no-return-assign": 2,
+    "no-self-compare": 2,
+    "no-shadow-restricted-names": 2,
+    "no-trailing-spaces": 2,
+    "no-undef": 2,
+    "no-unneeded-ternary": 2,
+    "no-unreachable": 2,
+    "no-unused-vars": [2, { "vars": "all", "args": "after-used" }],
+    "no-void": 2,
+    "no-with": 2,
+    "operator-linebreak": [2, "before"],
+    "padded-blocks": [2, "never"],
+    "quotes": [2, "single", "avoid-escape"],
+    "semi": [2, "always"],
+    "semi-spacing": [2, { "before": false, "after": true }],
+    "space-in-parens": [2, "never"],
+    "space-infix-ops": 2,
+    "space-return-throw-case": 2,
+    "space-unary-ops": [2, { "words": true, "nonwords": false }],
+    "use-isnan": 2,
+    "wrap-iife": 2
+  }
```

Comment:
Kinda screwed up here and just deleted any rule I wanted to disable; in reality we should leave them in the config. `--reset` is not yet the default, so most editors' linting plugins will re-enable those rules.